test/learn.jl

```diff
@@ -260,7 +258,7 @@ end
         plot_data = last(logger.logged["test_set_evaluation/metrics_per_epoch"])
```

Comment:
```suggestion
        plot_data = last(logger.logged["validation_set_evaluation/metrics_per_epoch"])
```

...to get the discrimination validation plot to show up in the grouped plots! (Somehow between when this plot and its group test were added and now, this change got lost---it isn't a regression due to this PR, but making this change will add slight coverage going forward!

Reply:
...this was totally a regression caused by me in #22, whoops! If only we'd had percy then....